### DISCUSSION TITLE
Hunting own bounty

### DISCUSSION THREAD

**Discussion by hanseartic** [kalepail/stellar-quest-bounties#97] [Q&A] [answered]
Is it ok for someone to first submit a bounty via the [dedicated bounty](https://github.com/tyvdh/stellar-quest-bounties/issues?q=label%3Acreate-bounties+label%3Abounty) and afterwards hunt _that_ new bounty themselves?

**Comment by ElliotFriend** [comment, marked as answer]
I know this was answered in the [discord](https://discord.com/channels/763798356484161566/869565771586605136/874700896519487538), but for the sake of completing the discussion here:

Yes. You can both submit, and subsequently hunt, your own bounty.
